apps/ember-test-app/app/components/f/form/select.gts

```diff
@@ -0,0 +1,154 @@
+  }
+
+  get objectOptionsSource() {
+    return JSON.stringify(this.objectOptions, null, 2);
+  }
+
+  <template>
+    <FreestyleSection @name="Select" as |Section|>
+      <Section.subsection @name="String Options">
+        <FreestyleUsage>
+          <:example>
+            <Select
+              @binding={{bind this "selectValue"}}
+              @disabled={{this.disabled}}
+              @loading={{this.loading}}
+              @scrollable={{this.scrollable}}
+              @options={{this.stringOptions}}
+            />
+          </:example>
+          <:api as |Args|>
+            <Args.Bool
+              @name="disabled"
+              @defaultValue={{false}}
+              @description="When true, the button will be disabled"
+              @value={{this.disabled}}
+              @onInput={{fn this.update "disabled"}}
+            />
+            <Args.Bool
+              @name="loading"
+              @defaultValue={{false}}
+              @description="When true, the text will be replaced with a loading spinner"
+              @value={{this.loading}}
+              @onInput={{fn this.update "loading"}}
+            />
+            <Args.Bool
+              @name="scrollable"
+              @defaultValue={{false}}
+              @description="Unless false, the dropdown will be scrollable"
+              @value={{this.scrollable}}
+              @onInput={{fn this.update "scrollable"}}
+            />
+          </:api>
+        </FreestyleUsage>
+      </Section.subsection>
+
+      <Section.subsection @name="Object Options">
+        <FreestyleUsage>
+          <:example>
+            <Select
+              @binding={{bind this "selectValue"}}
+              @disabled={{this.disabled}}
+              @loading={{this.loading}}
+              @scrollable={{this.scrollable}}
+              @options={{this.objectOptions}}
+              @displayPath="searchableDisplay"
+              @serializationPath="key"
+            />
+          </:example>
+        </FreestyleUsage>
+      </Section.subsection>
+
+      <Section.subsection @name="Yielded Options">
+        <FreestyleUsage>
+          <:example>
+            <Select
+              @binding={{bind this "selectValue"}}
+              @disabled={{this.disabled}}
+              @loading={{this.loading}}
+              @scrollable={{this.scrollable}}
+              @options={{this.objectOptions}}
+              @serializationPath="key"
+            >
+              <:empty>
+                <span>Nothing to see here</span>
+              </:empty>
+              <:display as |option|>
+                <span>Custom Display {{option.id}}</span>
+              </:display>
+              <:option as |option|>
+                <span>Custom Option {{option.id}}</span>
+              </:option>
+            </Select>
+          </:example>
+        </FreestyleUsage>
+      </Section.subsection>
+    </FreestyleSection>
+
+    <div class="grid">
+      <div class="g-col-4 border p-3">
```

Comment:
```suggestion
      <div class="g-col-4 border rounded p-3">
```

Reply:
On second thought, could you add the border to the `code-block` component? I think it'll help give more definition.